modules/security-context-constraints-about.adoc

```diff
@@ -31,8 +31,7 @@ If you have `cluster-admin` privileges, you can adjust the default SCC policies
 be more permissive.
 ////
 
-Docker has a
-link:https://docs.docker.com/engine/reference/run/#runtime-privilege-and-linux-capabilities[default list of capabilities]
+CRI-O has a link:https://github.com/cri-o/cri-o/blob/master/docs/crio.conf.5.md#crioruntime-table[default list of capabilities]
```

Comment:
Just wanted to check that this GitHub link has been approved for the docs by Eric (or whoever is applicable for approving this GitHub link)?

Reply:
Good question, that was on my list to ask after I got the initial SME review!

@sferich888 Are you okay with this GitHub link to list the default CRI-O capabilities? (Apparently the CRI-O default is different than the Docker default, which we previously linked to)

Reply:
No I don't think we want to link to upstream here. 
Our docs for a version of OCP are tied directly to versions of CRI-O (and the capabilities that version enables by default). If you link upstream then you could be saying (as an example) OCP 4.6 (CRI-O 1.19) capabilities are (see link) but because upstream is on CRI-O 1.21 (in development) you could be listing capabilities that aren't accurate for the version of CRI-O that OCP is using.

Reply:
Thanks, I'll list them out here instead.

Reply:
By adding the capabilities here, I'm adding more bullets to a section that already has too many bullets and text. I've opened https://issues.redhat.com/browse/OSDOCS-1898 to track an improvement effort for this section.